src/Maestro/maestro-angular/src/app/page/build-graph-table/build-graph-table.component.html

```diff
@@ -74,6 +74,7 @@
     </th>
     <th>Repository</th>
     <th>Date Produced</th>
+    <th><span title="Time it took for this dependency to flow to the root node in minutes">Time To Inclusion</span></th>
```

Comment:
I feel like this needs to be even more explicit about what is being measured.  Meaning, is the "time to inclusion" the time from when the dependency was produced to when the current build was produced?  Does "produced" means build completed or showed up in BAR?  

People are going to want to know, if I submit a change to A, is it available from a build of B in the "time to inclusion" metric value or do I also need to factor in build and push to BAR time of A?  Whatever the answer is, it's probably fine, but it needs to be clear what this metric is measuring.

Reply:
Made it more descriptive, but now I'm worried it's a little confusing?